From 43ef5a4f0f47192e8c03639cfe724afe8194bb26 Mon Sep 17 00:00:00 2001
From: Emmanuel Grumbach <emmanuel.grumbach@intel.com>
Date: Sun, 18 Jul 2021 15:07:38 +0300
Subject: [BUGFIX][NOUPSTREAM] iwlwifi: iwlmei isn't going to be merged
 anytime soon

One of the things that I learnt (the hard way) whie working upstream
is that you can't really expect when things are gonig to be merged.
So, instead of being disappointed every kernel release, let's forget
about this code and we might get a good surprise.
Another thing I learnt is that, usually, the surprises aren't good.
But the jewish people went through the centuries with the sole force
of the hope, so let's stick to our tradition.

type=bugfix
ticket=none
fixes=Ibf66e3e248d549166d1eb340c60ee5b2a1dada78

Change-Id: I2d107adba3744aedc6aa23bf4dfb69b2123d29bf
Signed-off-by: Emmanuel Grumbach <emmanuel.grumbach@intel.com>
Reviewed-on: https://git-amr-3.devtools.intel.com/gerrit/328956
automatic-review: ec ger unix iil jenkins <EC.GER.UNIX.IIL.JENKINS@INTEL.COM>
Tested-by: ec ger unix iil jenkins <EC.GER.UNIX.IIL.JENKINS@INTEL.COM>
Reviewed-by: Luciano Coelho <luciano.coelho@intel.com>
x-iwlwifi-stack-dev: 5b6122b574100f77d29635170b736430ba6f8d25
(backported from commit 87dce6e9817a64a51aa2719d7f8a080ae6a436a4)
Signed-off-by: You-Sheng Yang <vicamo.yang@canonical.com>
---
 drivers/net/wireless/intel/iwlwifi/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/intel/iwlwifi/Kconfig b/drivers/net/wireless/intel/iwlwifi/Kconfig
index da9e5935b..68d80e7dc 100644
--- a/drivers/net/wireless/intel/iwlwifi/Kconfig
+++ b/drivers/net/wireless/intel/iwlwifi/Kconfig
@@ -93,7 +93,7 @@ config IWLMEI
 	depends on INTEL_MEI
 	depends on IWLMVM
 	depends on IWLMVM_VENDOR_CMDS
-	depends on WLAN_VENDOR_INTEL_MEI || !KERNEL_5_15
+	depends on WLAN_VENDOR_INTEL_MEI || !KERNEL_5_20
 	help
 	  Enable V-PRO. This allows to communicate with the CSME firmware.
 	  This is supported starting from Tiger Lake platforms and has been
-- 
2.34.1

